src/plugins/visualize/public/application/components/visualize_listing.tsx

```diff
@@ -174,12 +174,6 @@ export const VisualizeListing = () => {
 
   return (
     <>
```

Comment:
Nice change! Now we can also remove this fragment

Reply:
Oh! Good catch! Have removed redundant fragments in this file.